calcolatrice.py

```diff
@@ -42,8 +42,9 @@ def exit_program():
         valore_finale = check_finale("sommare")
         check_null(valore_finale)
 
-        risultato = valore_iniziale + valore_finale
-        print(f"Il risultato della somma è: ", risultato)
+        if (valore_iniziale.isnumeric() & valore_finale.isnumeric()):
```

Comment:
Senza andare a modificare la struttura o l'approccio generale del programma, `.isnumeric()` e' una maniera quick di controllare se tutti i caratteri sono numerici e se c'e' almeno un carattere. [Qui](https://docs.python.org/3/library/stdtypes.html) puoi vedere la descrizione ed altre funzioni native che in base a cosa vuoi fare potrebbero essere piu' adatte.

Tieni a mente che gli input che stai memorizzando in `valore_iniziale` e `valore_finale` sono stringhe quindi se cambi il tipo `isnumeric()` non funzionera' perche' e' un metodo nativo del tipo string